dtao.c

```diff
@@ -423,33 +424,44 @@ event_loop(void)
 	int ret;
 	int wlfd = wl_display_get_fd(display);
 
-	fprintf(stderr, "%d\n", persist);
+	int tfd = 0;
+	struct itimerspec ts;
+	if (persist > 0) {
+		tfd = timerfd_create(CLOCK_REALTIME, 0);
```

Comment:
```diff
-		tfd = timerfd_create(CLOCK_REALTIME, 0);
+		tfd = timerfd_create(CLOCK_MONOTONIC, TFD_CLOEXEC);
```

Reply:
Did you try to create the timer even if persist <= 0?

Reply:
ah, forgot about the -1